CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Drafter Changelog
 
-## Master
+## 3.1.0-pre.0
```

Comment:
I think it's better if we do pre releases instead of pushing this out to the wild straight away. We've had many cases in the past where we have broken behaviour in Dredd and if we do a pre release this gives people the opportunity to test the pre release before it becomes a final release.